packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -398,6 +396,8 @@ export class ModularChangeFamily
 
 		if (change.fieldChanges !== undefined) {
 			modify.fields = this.intoDeltaImpl(change.fieldChanges, repairStore, path);
+		} else if (!Object.prototype.hasOwnProperty.call(modify, "setValue")) {
```

Comment:
Can't we just check `valueChange === undefined` instead?

Reply:
Or at the beginning of the function, return undefined if `change.valueChange === undefined && change.fieldChanges === undefined)?